src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
-'use strict';
+const but = document.querySelector('.logo');
 
-// write your code here
+const prom1 = new Promise((resolve) => {
+  but.addEventListener('click', () => {
+    const newDiv = document.createElement('div');
+
+    newDiv.innerHTML = 'Promise was resolved!';
+    newDiv.classList.add('message');
+
+    const parentElement = document.querySelector('body');
+
+    parentElement.appendChild(newDiv);
```

Comment:
you can simplify this logic

move all these lines to some helper function and call this function in `prom1.then(........)`